packages/vm/lib/state/stateManager.ts

```diff
@@ -161,7 +161,8 @@ export default class DefaultStateManager implements StateManager {
     const account = await this.getAccount(address)
     const storageTrie = this._trie.copy(false)
     storageTrie.root = account.stateRoot
-    storageTrie._checkpoints = []
+    //@ts-ignore
```

Comment:
just curious what this `@ts-ignore` is for, wondering if we could solve it without.

side note, i wish there was a finer grained `@ts-ignore {rule}` like `eslint-ignore-next-line {rule}`, would be much more informative :)

Reply:
Ah, this was still from the old trie version not being updated in Visual Studio Code (Visual Studio has its problems here or is it just me? 🤔 ) and I was blindly taking this over. So this is not necessary, will remove (next push). Thanks for the catch!